src/XrdCrypto/XrdCryptosslAux.cc

```diff
@@ -538,9 +569,6 @@ int XrdCryptosslX509ParseFile(const char *fname,
       }
    }
 
-   // We can close the file now
-   fclose(fcer);
-
```

Comment:
Where does this file get actually closed now?

Reply:
The caller owns the `FILE*` input and it's responsible for closing (as above).